ui/src/routes/app/[id]/store.svelte.ts

```diff
@@ -4,3 +4,4 @@ export const orderBy = $state({ v: 'LastUpdated' });
 export const limit = $state({ v: 50 });
 export const title = $state({ v: undefined });
 export const lastUpdated: { v: Date | undefined } = $state({ v: undefined });
+export const showImage = $state({ v: false });
```

Comment:
So, this store is used to communicate with the back end, since serving up a URL to steam's CDN is so cheap, it'll always be served up. This should be pushed into the main page.